dissect/target/helpers/mount.py

```diff
@@ -1,88 +1,103 @@
 import errno
 import logging
-from stat import S_IFDIR, S_IFREG
 from functools import lru_cache
+from typing import BinaryIO, Optional
 
 from fuse import FuseOSError, Operations
 
-from dissect.target import filesystem
+from dissect.target.filesystem import Filesystem, FilesystemEntry
 
 log = logging.getLogger(__name__)
 
 
 class DissectMount(Operations):
-    def __init__(self, fs):
+    def __init__(self, fs: Filesystem):
         self.fs = fs
+        self.file_handles: dict[int, BinaryIO] = {}
+        self.dir_handles: dict[int, FilesystemEntry] = {}
 
-    def _get(self, path):
+    @lru_cache(1024 * 1024)
+    def _get(self, path: str) -> FilesystemEntry:
         try:
             return self.fs.get(path)
         except Exception:
             raise FuseOSError(errno.ENOENT)
 
-    @lru_cache(4096)
-    def getattr(self, path, fh=None):
-        log.debug("fuse::getattr(%s, %s)", path, fh)
-
+    @lru_cache(1024 * 1024)
```

Comment:
It would be called `ARBITRARY_HIGH_NUMBER` to be honest :smile:

Reply:
Even so, it might be good to communicate this because otherwise people might be reluctant to change it because they assume there is a specific reason for this expression.